content/hi/continuous_integraion.md

```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर एकीकरण
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर एकीकरण (Continuous_Integration) , जिसे  अक्सर CI कहा जाता है, कोड परिवर्तन को जल्दी से जल्दी एकीकृत करने का एक अभ्यास है। CI, [CD](/Continuous_Delivery/) पढ़ने के लिए आवश्यक है। ऐतिहासिक रूप से, CI की प्रक्रिया तब शरू होती है जब कोड परिवर्तन को सोर्स नियंत्रण सिस्टम (Git, Mercurial, या Subversal) में कमिट किया जाता है और वह परीक्षित उत्पाद, CD सिस्टम में जाने के लिए तैयार होता है।

```

Comment:
I feel like we shouldn't change the topic name to `निरंतर एकीकरण` as continuous Integration is a technical term

Reply:
Yeah, the `topic:`

Reply:
ok

Reply:
Done